src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.cs

```diff
@@ -40,7 +40,7 @@ internal sealed partial class CalendarData
         internal string[] saLeapYearMonthNames = null!; // Multiple strings for the month names in a leap year.
 
         // Integers at end to make marshaller happier
-        internal int iTwoDigitYearMax = 2029; // Max 2 digit year (for Y2K bug data entry)
+        internal int iTwoDigitYearMax = 2049; // Max 2 digit year (for Y2K bug data entry)
```

Comment:
I'm not sure what `(for Y2K bug data entry)` means but it might be a very obsolete comment at this point. I'd remove it

Reply:
Sure thing, this has been removed

Reply:
I still see it here and below :)

Reply:
`for Y2K bug data entry` was a helper comment telling how important the two digit year max for Y2K problem when mostly two digit years was used than four digit years.

Reply:
> I still see it here and below :)

Whoops--I mixed up the comments and thought this was talking about the .sln file changes. The Y2K note on line 43 has been removed